main.js

```diff
@@ -77,4 +80,32 @@ window.addEventListener('DOMContentLoaded', Methods.addLS);
 addBook.addEventListener('click', (e) => {
   e.preventDefault();
   Methods.addUI();
-});
\ No newline at end of file
+});
+
+// select sections
+const viewBooks = document.querySelector('#book-list');
+const viewAddBook = document.querySelector('#add-new');
+const vieContact = document.querySelector('#contact');
+//
+const displayBookSec = document.querySelector('.container');
+const fieldAddSec = document.querySelector('.field-add');
+const contactSec = document.querySelector('.contact');
+// add event listners
+viewBooks.addEventListener('click', () => {
+  viewBooks.style.display = 'block';
+  displayBookSec.style.display = 'flex';
+  fieldAddSec.style.display = 'none';
+  contactSec.style.display = 'none';
+});
+
+viewAddBook.addEventListener('click', () => {
+  fieldAddSec.style.display = 'flex';
+  displayBookSec.style.display = 'none';
+  contactSec.style.display = 'none';
+});
```

Comment:
- [ ] (Optional) A good User Interaction feature would be to inform the user if his book has been added and direct the user to click the link on the NavBar that displays all books or you change the view to the view book section after you have added the link. 👍

Reply:
Thank you for providing your feedback on this repository, I really appreciate it. 👍
according to the recommendations that you gave I've done the following changes :
> Added the padding-bottom to the contacts section so that the footer doesn't cover the contacts section
> Added a function that displays a message whenever the user is adding a book with an empty input field and displays a success when the user does the opposite and redirects the user when clicking that message.

Reply:
Great work @NabilHY. Continue the good work and you will achieve great success.